auth-service/build.gradle.kts

```diff
@@ -58,6 +58,9 @@ dependencies {
     implementation("io.ktor:ktor-server-metrics-micrometer:$ktorVersion")
     implementation("io.micrometer:micrometer-registry-cloudwatch2:1.11.2")
 
+    //Emails
+    implementation("com.sun.mail:javax.mail:1.6.2")
```

Comment:
https://mvnrepository.com/artifact/com.sun.mail/jakarta.mail is the modern version